### DISCUSSION TITLE
CR-200b Pro source (Marlin)

### DISCUSSION THREAD

**Discussion by TrueLabInc** [CrealityOfficial/Creality_Sonic_Pad#106] [Ideas]
Hi all!
Any ideas on how to get the Cr-200b Pro main board sources? 
I found several shortcomings, one of the most unpleasant is that the filament cooling fan only works in 0 or 100% mode, i.e. on/off even if in gcode M106 S1, any value greater than 0 results in 100% fan power. I have the latest 1.0.6 firmware. Creality support has not been providing any results for many days now.
![изображение_viber_2024-04-02_09-24-58-464](https://github.com/CrealityOfficial/Creality_Sonic_Pad/assets/53824824/515429ed-ad83-40a4-9c95-4abe0ba52cdd)

**Comment by ReneHezser** [comment]
If I am not mistaken, the configuration has to be made available, as the Marlin license requires this.
https://github.com/MarlinFirmware/Marlin/blob/bugfix-2.1.x/LICENSE#L38
